packages/backend/src/tests/getTableTypes.ts

```diff
@@ -0,0 +1,9 @@
+export type TableTypes<Table> = {
+    [TTable in keyof Table]: Table[TTable] extends object
```

Comment:
I think this type can be simplified: do you really need extends object and then extends { type: infer C }?

Also `C` is a bad name. Prefer the convention T<DescriptiveName>